Apps/frontend/src/views/AmplitudeSlider.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+  import { amplitudeAdjustment } from "../stores.js";
+  import { MIN_AMPLITUDE, MAX_AMPLITUDE } from "../const.js";
+  export let onInput = (amplitude) => {
```

Comment:
Would prefer an error if oInput is not defined (referring to OffsetSlider.svelte's onInput)

Reply:
thanks! changed it.